controllers/clustermanager_controller_test.go

```diff
@@ -0,0 +1,217 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	enterpriseApi "github.com/splunk/splunk-operator/api/v4"
+
+	"time"
+
+	"github.com/splunk/splunk-operator/controllers/testutils"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+var _ = Describe("ClusterManager Controller", func() {
+
+	BeforeEach(func() {
+		time.Sleep(2 * time.Second)
+	})
+
+	AfterEach(func() {
+
+	})
+
+	Context("ClusterManager Management failed", func() {
+
+		It("Get ClusterManager custom resource should failed", func() {
```

Comment:
nit: should fail

Reply:
This is coming from develop. https://github.com/splunk/splunk-operator/blob/develop/controllers/clustermanager_controller_test.go#L35